apps/homework/models.py

```diff
@@ -162,12 +183,20 @@ class Question(models.Model):
     has_specific_answer = models.BooleanField(default=False)
 
     question = models.CharField(max_length=300)
-    answer = models.CharField(max_length=200, null=True, blank=True)
+    answer = models.TextField(blank=True, default='')
 
     def __str__(self):
         return self.question
 
 
+class QuestionAnswer(models.Model):
```

Comment:
This was just moved correct?

Reply:
Yes. There are model changes in the Forms branch, but not here.